hooks/playbooks/run_tofu.yml

```diff
@@ -22,8 +22,8 @@
     - name: Create openstack config dir
       ansible.builtin.file:
         path: "{{ ansible_user_dir }}/.config/openstack/"
-        owner: zuul
-        group: zuul
+        owner: "{{ ansible_user_id }}"
```

Comment:
IIRC, `ansible_user_id` is the name of the user which is executing the playbook, where `ansible_user` is the username on which you ssh to the remote host, right?
Wondering if `ansible_user` would be not enough here.